packages/driver/cypress/integration/e2e/multi-domain/multi_domain_spec.ts

```diff
@@ -1,75 +1,5 @@
 // @ts-ignore / session support is needed for visiting about:blank between tests
 describe('multi-domain', { experimentalSessionSupport: true }, () => {
-  beforeEach(() => {
-    cy.visit('/fixtures/multi-domain.html')
-    cy.get('a[data-cy="multi-domain-secondary-link"]').click()
-  })
-
-  it('runs commands in secondary domain', () => {
-    cy.switchToDomain('http://foobar.com:3500', () => {
-      cy
-      .get('[data-cy="dom-check"]')
-      .invoke('text')
-      .should('equal', 'From a secondary domain')
-    })
-
-    cy.log('after switchToDomain')
-  })
-
-  it('passes runnable state to the secondary domain', () => {
-    const runnable = cy.state('runnable')
-    const expectedRunnable = {
-      clearTimeout: null,
-      isPending: null,
-      resetTimeout: null,
-      timeout: null,
-      id: runnable.id,
-      _currentRetry: runnable._currentRetry,
-      _timeout: 4000,
-      type: 'test',
-      title: 'passes runnable state to the secondary domain',
-      titlePath: [
-        'multi-domain',
-        'passes runnable state to the secondary domain',
-      ],
-      parent: {
-        id: runnable.parent.id,
-        type: 'suite',
-        title: 'multi-domain',
-        titlePath: [
-          'multi-domain',
-        ],
-        parent: {
-          id: runnable.parent.parent.id,
-          type: 'suite',
-          title: '',
-          titlePath: undefined,
-          ctx: {},
-        },
-        ctx: {},
-      },
-      ctx: {},
-    }
-
-    cy.switchToDomain('http://foobar.com:3500', [expectedRunnable], ([expectedRunnable]) => {
-      const actualRunnable = cy.state('runnable')
-
-      expect(actualRunnable.titlePath()).to.deep.equal(expectedRunnable.titlePath)
-      expectedRunnable.titlePath = actualRunnable.titlePath
-
-      expect(actualRunnable.title).to.equal(expectedRunnable.title)
-      expect(actualRunnable.id).to.equal(expectedRunnable.id)
-      expect(actualRunnable.ctx).to.deep.equal(expectedRunnable.ctx)
-      expect(actualRunnable._timeout).to.equal(expectedRunnable._timeout)
-      expect(actualRunnable.type).to.equal(expectedRunnable.type)
-      expect(actualRunnable.callback).to.exist
-      expect(actualRunnable.timeout).to.exist
-      expect(actualRunnable.parent.title).to.equal(expectedRunnable.parent.title)
-      expect(actualRunnable.parent.type).to.equal(expectedRunnable.parent.type)
-      // expect(actualRunnable).to.deep.equal(expectedRunnable)
-    })
-  })
-
   it('passes viewportWidth/Height state to the secondary domain', () => {
     const expectedViewport = [320, 480]
```

Comment:
The test changes in here are related to  not being able to call cy.viewport between a click that switches domains in the beforeEach and the switchToDomain command in the test.